src/client/swagger/managedFunctions.yml

```diff
@@ -60,21 +60,21 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/api/v2private/functions/trigger':
+  '/api/v2/functions/trigger':
     post:
-      operationId: PostFunctionTrigger
+      operationId: PostFunctionsTrigger
       tags:
         - Functions
-      summary: Manually trigger a function execution without creating an association function resource
+      summary: Manually trigger a function without creating an associated function resource
       requestBody:
-        description: Function to run
+        description: Function to be triggered
         required: true
         content:
           application/json:
             schema:
               $ref: '#/components/schemas/FunctionTriggerRequest'
       responses:
-        '200':
+        '201':
```

Comment:
Just a nitpick here.

I am not sure this should be a 201 as we are not creating anything. It would make sense that `

If we may want to make it 201 just to be consistent with other APIs, I am not strong on using 200 here, but 201 does not seem natural as nothing would get created in our CRUD APIs (as this is not a stored function, no logs or runs of that trigger would get stored anywhere).

It would make sense that POST to `/api/v2/functions/{functionID}/truns` would return 201 since that API creates a new run and this is also what tasks API returns when you manually run a task. But I think we've removed this API.

https://tools.ietf.org/html/rfc7231#section-6.3.2

Reply:
That does make sense, thanks!